src/plugins/DocManager/DocManager.js

```diff
@@ -149,8 +149,19 @@ class DocManager extends IDEPlugin {
    * @param {{name: String, scope: String}} modelKey
    */
   reloadDoc(modelKey) {
-    const { name, scope } = modelKey;
-    return this.getStore(scope)?.loadDoc(name);
+    return this.call(
+      PLUGINS.DOC_MANAGER.NAME,
+      PLUGINS.DOC_MANAGER.CALL.READ,
+      modelKey
+    ).then(doc => this.call(
```

Comment:
Maybe you don't need to call the doc_manager plugin inside the document manager

Reply:
True :)